scripts/oneoff/mark-g12-supplier-status.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+"""
+We have one supplier in the G12 recovery process who has not yet signed the framework agreement. In order for them
+to do this, we need to evaluate their answers to the declaration questions and mark them as being on G-Cloud 12 if
+they pass.
+
+This script borrows heavily from
+digitalmarketplace-scripts/blob/master/scripts/framework-applications/mark-definite-framework-results.py
+to do so for this single supplier.
+
+Usage: mark-g12-supplier-status.py <stage> --updated-by=<updater> --dry-run
+"""
+import sys
+import logging
+import json
+from docopt import docopt
+from dmapiclient import DataAPIClient
+from dmutils.env_helpers import get_api_endpoint_from_stage
+sys.path.insert(0, ".")
+
+from dmscripts.mark_definite_framework_results import _passes_validation, pass_supplier
+from dmscripts.helpers.auth_helpers import get_auth_token
+
+UNSIGNED_SUPPLIER_ID = 712034
```

Comment:
I guess more typically this would be an argument to the script but since this should be really and once ever thing, seems OK to hardcode.

Reply:
Yes, I definitely agree. My thinking here was that I didn't want to run the risk of passing the wrong ID as an argument since we're doing something a bit weird